src/foam/u2/view/ReciprocalSearch.js

```diff
@@ -99,11 +99,17 @@ foam.CLASS({
     },
     {
       class: 'Int',
-      name: 'selectedCount'
+      name: 'selectedCount',
+      postSet: function() {
```

Comment:
I think you forgot to ditch these postSets

Reply:
Thanks. Fixed by 93b99e5.